second-edition/src/ch12-01-accepting-command-line-arguments.md

```diff
@@ -93,22 +93,22 @@ $ cargo run needle haystack
 ["target/debug/minigrep", "needle", "haystack"]
 ```
 
-Notice that the first value in the vector is `"target/debug/minigrep"`, which
-is the name of our binary. This matches the behavior of the arguments list in
-C, letting programs use the name by which they were invoked in their execution.
-It’s often convenient to have access to the program name in case we want to
-print it in messages or change behavior of the program based on what command
-line alias was used to invoke the program. But for the purposes of this
-chapter, we’ll ignore it and save only the two arguments we need.
+Nota que el primer valor en el vector es `"target/debug/minigrep"`, que es el
+nombre de nuestro binario. Esto coincide con el comportamiento de la lista de argumentos
+en C, permitiendo a los programas usar el nombre con el que fueron invocados en su ejecución.
+A menudo es conveniente tener acceso al nombre del programa en caso de que queramos 
+imprimirlo en mensajes o cambiar el comportamiento del programa basado en el alias de la
+línea de comandos que se usó para invocar el programa. Pero para los propósitos de este
+capítulo, lo ignoraremos y sólo guardaremos los dos argumentos que necesitamos.
 
-### Saving the Argument Values in Variables
+### Guardando los Valores Argumentales en Variables
 
-Printing the value of the vector of arguments illustrated that the program is
-able to access the values specified as command line arguments. Now we need to
-save the values of the two arguments in variables so we can use the values
-throughout the rest of the program. We do that in Listing 12-2:
+Al imprimir el valor del vector de argumentos se ilustró que el programa puede
+acceder a los valores especificados como argumentos de línea de comandos. Ahora necesitamos 
+guardar los valores de los dos argumentos en variables para que podamos usar los valores
+en el resto del programa. Lo hacemos en el listado 12-2:
 
-<span class="filename">Filename: src/main.rs</span>
+<span class="filename">Nombre del archivo: src/main.rs</span>
 
 ```rust,should_panic
 use std::env;
```

Comment:
Please translate the comments on the lines below (inside println)

Reply:
all done :)